apis/infrastructure/v1beta1/byomachine_types.go

```diff
@@ -50,6 +56,10 @@ type ByoMachineStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
 
+	// HostInfo has the attached host platform details.
+	// +optional
+	HostInfo HostInfo `json:"hostinfo,omitempty"`
```

Comment:
I am assuming this is part of #470 ?

Reply:
The same info needs to be present in `ByoMachine` as well for the `k8sInstallerConfigController` to consume.

Reply:
Why same info? Can't we read it from ByoHost's object?

Reply:
We did discuss this. The `ByoMachine` will not a have a reference to the attached `ByoHost`. We will have 2 choices
- Add `ByoHost` Reference in `ByoMachine` CRD, and `k8sInstaller` will first get the `ByoMachine` and then `ByoHost` to fetch these details.
- Add the Host details in the `ByoMachine`

Reply:
Okay, makes sense. Thanks!